draft-demarco-nonce-endpoint.md

```diff
@@ -68,7 +68,9 @@ The **Nonce Endpoint** MUST satisfy the following requirements:
 
 The **Nonce** MUST satisfy the following requirements:
 
-- Pure opacity to recieving Clients.
+- It MUST be opaque to receiving Clients.
+- It MUST be unguessable, by cryptographic randomness or encryption.
```

Comment:
if we can avoid the "or" here, lots of implementation complexity will go away.

Reply:
see https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/17/files#r1481755422